diff --git a/node_modules/react-leaflet/lib/MapContainer.js b/node_modules/react-leaflet/lib/MapContainer.js
index 2b21760..7fdf96b 100644
--- a/node_modules/react-leaflet/lib/MapContainer.js
+++ b/node_modules/react-leaflet/lib/MapContainer.js
@@ -16,11 +16,11 @@ import { LeafletProvider, createLeafletContext } from '@react-leaflet/core';
 import { Map as LeafletMap } from 'leaflet';
 import React, { forwardRef, useCallback, useEffect, useImperativeHandle, useState } from 'react';
 function MapContainerComponent({ bounds , boundsOptions , center , children , className , id , placeholder , style , whenReady , zoom , ...options }, forwardedRef) {
-    const [props] = useState({
-        className,
-        id,
-        style
-    });
+    // const [props] = useState({
+    //     className,
+    //     id,
+    //     style
+    // });
     const [context, setContext] = useState(null);
     useImperativeHandle(forwardedRef, ()=>context?.map ?? null, [
         context
@@ -50,7 +50,11 @@ function MapContainerComponent({ bounds , boundsOptions , center , children , cl
     const contents = context ? /*#__PURE__*/ React.createElement(LeafletProvider, {
         value: context
     }, children) : placeholder ?? null;
-    return /*#__PURE__*/ React.createElement("div", _extends({}, props, {
+    return /*#__PURE__*/ React.createElement("div", _extends({}, {
+        className,
+        id,
+        style
+    }, {
         ref: mapRef
     }), contents);
 }
